ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVUtilsTransactionSpec.scala

```diff
@@ -107,6 +107,40 @@ class KVUtilsTransactionSpec extends WordSpec with Matchers {
       }
     }
 
+    "pre-compute blinding info when submitting a transaction" in KVTest.runTestWithSimplePackage(
```

Comment:
Consider merging these two new test cases with the above (they only differ in 1 line). I.e., if pre-computing blinding info is a must when we submit a transaction then the assertion can be in the same test case.

Reply:
Done.